Git-Workflow-Guidance.md

```diff
@@ -557,14 +666,54 @@ If you want a useful graphical summary of the log, try
  | merge | Merge changes from local repo to sandbox |
  | pull | Download changes to local sandbox (== fetch + merge) |
  | push | Upload changes |
- | rebase | like pull but updates the base version then replays local changes |
+ | rebase | like pull but updates the base version then replays local changes, same as pull --rebase |
  | remote [-v] [add] | 	Lists/Sets remote repositories in a sandbox |
  | rm | Removes a file from the repo |
  | status    | Provides summary of status of sandbox |
  | submodule | 	Many options, useful for working with submodules |
  | tag -a | Create a tag or list tags |
  | update | Pull updates into the local sandbox |
 
+### Further Information
+
+There are many git commands to diagnose the status of your sandbox.
+
+#### Status
+Status provides a summary of the status of your sandbox.  
+
+      git status
+
+will indicate which branch your sandbox is on and the changes that have been made.  This should be used often during development and before commits and pushes.
+
+#### Diff
+Diff shows code changes.
+
+      git diff
+
+If you also want to see the submodule diffs, use
+
+      git diff --submodule
+
+but it's important to remember that a commit to CICE will NOT commit the Icepack changes to Icepack.
```

Comment:
Again, I feel this could confuse folks and lead to a conclusion that these Icepack diffs are somehow part of CICE.  And I'm still not convinced we should do this.  If we emphasize development of CICE and Icepack separately and try to keep them separate as much as possible and downplay all the submodule stuff, I think it's easier.

Reply:
OK... then I guess we should phrase it as "If you also want to see the submodule changes" as I suggested above.